docs/workflow-actor.md

```diff
@@ -0,0 +1,28 @@
+# Workflow Actor
+
+> v3.6.5 and after
```

Comment:
I think you'll need to add this here to make the spellchecker happy: https://github.com/argoproj/argo-workflows/blob/f1722947cb177b711c62eaa6cd292c812818480e/.spelling#L259

Reply:
Thanks